### ISSUE TITLE
Is one learning rate adequate for both models' convergence?

### ISSUE THREAD

**Issue by dexter2406** [nianticlabs/monodepth2#337]
In the code, the `learning rate=1e-4` is set to be constant for 15 epochs. Is there any other flexible recommendation on the value settings, like cosine, exponential? Or even each model could use its own learning rate?

Say we have a *pose* encoder-decoder that has been trained for 10 epochs. But the *depth* nets are from scratch. Wouldn't `lr=1e-4` be too large for the pose nets? Since the loss is based on the images, so the loss won't  decrease as long as the depth nets are not good enough.

Also, I wonder whether we need to adopt a different (larger) learning rate, if we train encoders with scratch (don't use ImageNet pretrained weights)?

**Comment by mdfirman** [comment]
I would comment on this that: Yes, you could probably get much better results by using alternative approaches to learning rate adjustment. But this sort of hyperparameter tuning wasn't the focus of our paper, so we used something pretty simple instead.

If training from scratch then yes I could imagine using a higher learning rate (or perhaps just training for longer at the current learning rates) might improve things.

Sorry I don't have a very concrete answer here, but it's just not something we are able to guess about.
